platform/darwin/src/http_file_source.mm

```diff
@@ -199,6 +199,7 @@ BOOL isValidMapboxEndpoint(NSURL *url) {
     
     NSURL *url = [NSURL URLWithString:@(resource.url.c_str())];
 
+#if TARGET_OS_IPHONE || TARGET_OS_SIMULATOR
```

Comment:
Would you mind moving this guard so that `offline=true` gets added but `sku` does not? It isn’t critical, but offline pack downloading is one of the main things people are using the macOS map SDK for these days, so it would be beneficial for that to come through in requests just like on iOS.